### PR TITLE
Agent log config

### PR DESCRIPTION
**Pull request by jahkeup**
### Summary

This PR extends the changes in #192 to include environment variables to change the values of the logging driver options used for the supervised container.

### Implementation details

Environment variables are loaded from `/etc/ecs/ecs.config`, specifically:

- `ECS_INIT_DOCKER_LOG_FILE_SIZE`
- `ECS_INIT_DOCKER_LOG_FILE_NUM`

### Testing

New tests cover the changes: no

### Description for the changelog

Set default logging config for Agent container to reduce disk usage.

### Licensing

This contribution is under the terms of the Apache 2.0 License: yes

### CODE REVIEW

**Comment by issei-m** [comment]
Any plans to ship this? I'm facing on the problem where disk space is exhausted by the log 😭

**Comment by jahkeup** [comment]
I'm sorry to hear that, @issei-m, we do know this is needed and I will be getting back to complete this PR this this week.

**Comment by jahkeup** [comment]
@adnxn can you take a look?

File: ecs-init/config/common.go [hunk 1]
```diff
@@ -45,6 +46,23 @@ const (
 	// DefaultRegionName is the default region to fall back if the user's region is not a region containing
 	// the agent bucket
 	DefaultRegionName = endpoints.UsEast1RegionID
+
+	// dockerJSONLogMaxSize is the maximum allowed size of the
+	// individual backing json log files for the managed container.
+	dockerJSONLogMaxSize = "16m"
```
Comment (petderek) [inline ecs-init/config/common.go]: whats the current size?

Comment (jahkeup) [reply ecs-init/config/common.go]: There isn't a limit on the file size by default:

The option's default value is `-1`:
https://github.com/docker/docker-ce/blob/a70ecfc169d3149fba0392c8b26cb5ed1aecc6e3/components/engine/daemon/logger/jsonfilelog/jsonfilelog.go#L45-L55

Which means that the file won't be limited by the underlying utility:
https://github.com/docker/docker-ce/blob/a70ecfc169d3149fba0392c8b26cb5ed1aecc6e3/components/engine/daemon/logger/loggerutils/logfile.go#L169-L172

**Pull request merged**